git_machete/generated_docs.py

```diff
@@ -478,8 +482,8 @@
         Branches `adjust-reads-prec`, `edit-margin-not-allowed` and `grep-errors-script` are direct downstream branches for `develop`.
         `block-cancel-order` is a downstream branch of `adjust-reads-prec`, `change-table` is a downstream branch of `block-cancel-order` and so on.
 
-        Every branch name can be followed (after a single space as a delimiter) by a custom annotation — a PR number in the above example.
-        The annotations don't influence the way `git machete` operates other than that they are displayed in the output of the `status` command.
+        Every branch name can be followed (after a single space as a delimiter) by a custom annotation, e.g. PR #234, PR #235 rebase=no or push=no.
```

Comment:
Maybe put the example annotations in double-backticks in RST, so that they're single-backticked here in generated docs